### DISCUSSION TITLE
floccus changing bookmark names that were previously empty

### DISCUSSION THREAD

**Discussion by m4rtini2** [floccusaddon/floccus#1145] [Q&A]
Hello,

i have some bookmarks in my bookmark bar in firefox that have no name. There is just the favicon. After a sync floccus has put the url in the name of some of them but not in all and now they don't fit into the bar. This happens also on other computer after a sync. 
How can i tell floccus not to modify my bookmarks? How do i tell floccus to reverse this change?

**Comment by marcelklehr** [comment]
I think this was fixed in the meantime. Can you confirm whether this is still an issue?

**Reply by patienttruth** [reply]
@marcelklehr I'm having this issue. I haven't been able to pin down if it's Brave(Brave 1.84.141 (Official Build) (64-bit) Chromium: 142.0.7444.176), Floccus (5.7.0), or Nextcloud (Nextcloud Hub 10 (31.0.10)), but I've customized a few names, and they keep getting changed to the actual title of the webpage at whatever time the change happens. Eg. I just had a local server down yesterday and I have a bookmark called "502 Bad Gateway" though the favicon still reflects the actual server.

**Reply by marcelklehr** [reply]
Hi! I'm sorry for the delay. Is it possible that the bookmark in question previously had an empty title?

**Reply by marcelklehr** [reply]
This would happen in older bookmarks app versions, but should no longer happen. Which version of the bookmarks app do you have installed in Nextcloud?

**Reply by patienttruth** [reply]
It's all good!

Currently 16.0.1, however I'm not sure of any recent updates.

It's possible the website has no title (I'd have to confirm), but I do manually set the title, sync it, and then at some future time it changes.

*Now that I think about it, the second case I had was my Immich server which definitely has a page title. The first case is the Washington Sales Tax Lookup page, and it reverts to its URL.

**Reply by patienttruth** [reply]
And ... My bad, I missed the detail on the OP about the name being empty. So maybe mine is a different issue.
